aten/src/ATen/native/native_functions.yaml

```diff
@@ -2130,6 +2130,14 @@
   use_c10_dispatcher: full
   variants: method  # This is method-only to match the previous tensor API. In the future we could make this a function too.
 
+- func: moveaxis(Tensor self, int[] src, int[] dst) -> Tensor
+  use_c10_dispatcher: full
+  variants: function
+
+- func: moveaxis.int(Tensor self, int src, int dst) -> Tensor
+  use_c10_dispatcher: full
+  variants: function
```

Comment:
These should both have overload names. We can name the first one `moveaxis.intlist`

Reply:
Done.